package.json

```diff
@@ -11,6 +11,7 @@
     "update-references": "update-ts-references",
     "prettier-check": "prettier --check \"**/*.{js,ts,tsx,md,mdx,svelte}\"",
     "type-check": "tsc --build",
+    "type-check:full": "tsc --build tsconfig.with-examples.json",
```

Comment:
which one will run in CI? important that we keep full checks there

Reply:
i've updated it to run `type-check:full` in CI. I made type-checking faster in the IDE that uses the default tsconfig (including examples in it makes it way slower)